texar/torch/modules/networks/conv_networks.py

```diff
@@ -197,9 +224,15 @@ def default_hparams():
                layers will have the same stride. If a list, the list length
                must equal ``"num_conv_layers"``.
 
-           `"other_pool_kwargs"`: dict, optional
+           `"other_pool_kwargs"`: list or dict, optional
                Other keyword arguments for pooling layer class constructor.
 
+               - If a dict, the same dict is applied to all the pooling layers.
+
+               - If a list, the length must equal ``"num_conv_layers"``. The
```

Comment:
Can this (or is it necessary to) be a nested list as w/ `other_conv_kwargs`?

Reply:
Currently, this cannot be a nested list. The same holds true for `pool_size` and `pool_stride`